docs/tutorials/002_Parcel_Class.ipynb

```diff
@@ -0,0 +1,549 @@
+    "\n",
+    "# let's look at the first three entries here too\n",
+    "print('radii of particles in the parcel:', all_the_units_parcel.particle_radii_list()[0:3])\n",
+    "print('density of particles in the parcel:', all_the_units_parcel.particle_densities_list()[0:3])\n",
+    "print('charges of particles in the parcel:', all_the_units_parcel.particle_charges_list()[0:3])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets say we forgot to input all the units, what does that look like?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "radii of particles in the parcel: [<Quantity(5e-08, 'meter')>, <Quantity(5.95959596e-08, 'meter')>, <Quantity(6.91919192e-08, 'meter')>]\n",
+      "density of particles in the parcel: [<Quantity(1.0, 'kilogram / meter ** 3')>, <Quantity(1.17171717, 'kilogram / meter ** 3')>, <Quantity(1.34343434, 'kilogram / meter ** 3')>]\n",
+      "charges of particles in the parcel: [<Quantity(1.0, 'dimensionless')>, <Quantity(1.0, 'dimensionless')>, <Quantity(1.0, 'dimensionless')>]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# create a new parcel\n",
+    "missing_units_parcel = parcel.Parcel('many_particles', standard_environment)\n",
+    "\n",
+    "# create an array of radii\n",
+    "radii_array = np.linspace(50, 1000, num=100) * u.nm # units assigned are nanometer\n",
+    "densities_array = np.linspace(1, 18, num=100) # no units assigned\n",
+    "charges_array = np.ones(100) # no units assigned\n",
+    "\n",
+    "# adding many particles to the parcel, with specific radii units but no units for densities, and charges.\n",
+    "missing_units_parcel.create_and_add_list_of_particle('p_list', radii_array, densities_of_particles=densities_array, charges_of_particles=charges_array) # units are converted to [m], [kg/m^3], [dimensionless]\n",
+    "\n",
+    "# let's look at the first three entries here too\n",
+    "print('radii of particles in the parcel:', missing_units_parcel.particle_radii_list()[0:3])\n",
+    "print('density of particles in the parcel:', missing_units_parcel.particle_densities_list()[0:3])\n",
+    "print('charges of particles in the parcel:', missing_units_parcel.particle_charges_list()[0:3])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output looks ok, except for very low density particles due to the default units assumed. However, the trouble comes in when we try to calculate the particle coagulation properties. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "DimensionalityError",
+     "evalue": "Cannot convert from 'nanometer ** 3' ([length] ** 3) to 'kilogram' ([mass])",
```

Comment:
Solved. Within `particle.py`, the units enforcement/assignment for radius and density occur when those methods are called. So even though `missing_units_parcel` was given no units for its density values, calling the `density` method will assign and output the default units. 

The problem arises when the mass assignment happens using the unitless inputs. Here `density`, taken straight from the input, has no units, and causes an error. 
`self._mass = density() * (4*np.pi/3) * (radius()**3)`

If instead we take it from the method calls `self.density()`, then it will calculate mass with no issues:
`self._mass = self.density() * (4*np.pi/3) * (self.radius()**3)`